test/e2e/far_e2e_test.go

```diff
@@ -277,3 +310,18 @@ func wasNodeRebooted(nodeName string, nodeBootTimeBefore time.Time) {
 
 	log.Info("successful reboot", "node", nodeName, "offset between last boot", nodeBootTimeAfter.Sub(nodeBootTimeBefore), "new boot time", nodeBootTimeAfter)
 }
+
+// remediateNode run three functions to verify whether the node was remediated
+func remediateNode(nodeName, logString string, nodeBootTimeBefore time.Time) {
+	By("Adding FAR NoExecute taint")
```

Comment:
misleading comment, as far as I can tell we do not add the taint, but check it was added?